packages/trader/src/App/Components/Elements/Errors/error-component.jsx

```diff
@@ -1,17 +1,33 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { PageError } from '@deriv/components';
+import { PageError, Dialog } from '@deriv/components';
 import { routes } from '@deriv/shared';
-import { Localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
 
-const ErrorComponent = ({ header, message, redirect_label, redirectOnClick, should_show_refresh = true }) => {
+const ErrorComponent = ({
+    header,
+    message,
+    is_dialog,
+    redirect_label,
+    redirectOnClick,
+    should_show_refresh = true,
+}) => {
     const refresh_message = should_show_refresh ? (
         <Localize i18n_default_text='Please refresh this page to continue.' />
     ) : (
         ''
     );
 
-    return (
+    return is_dialog ? (
+        <Dialog
+            title={localize('There was an error')}
+            is_visible
+            confirm_button_text={localize('Ok')}
```

Comment:
shouldn't we show `redirect_label` as button text?
```suggestion
            confirm_button_text={redirect_label || localize('Ok')}
```

Reply:
@balakrishna-binary Regards the design, it is only `Ok` that's why I didn't set anything else. so according to that, should I add `redirect_label` as well?

Reply:
@shayan-binary-2 we can pass `Ok` as `redirect_label` prop from `contract.jsx`. I guess it's better to pass title and button text from the parent component so that `ErrorComponent` is more customizable.

Reply:
@balakrishna-binary thank you, just updated the code